src/SFA.DAS.AANHub.Api.UnitTests/Models/CreateApprenticeMemberCommandTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Api.Models;
+using SFA.DAS.AANHub.Application.Apprentices.Commands;
+using SFA.DAS.AANHub.Application.UnitTests;
+
+namespace SFA.DAS.AANHub.Api.UnitTests.Models
+{
+    [TestFixture]
+    public class CreateApprenticeMemberCommandTests
+    {
+        [Test, AutoMoqData]
+        public void Operator_PopulatesModelFromSummaryModel(CreateApprenticeModel createApprenticeModel)
+        {
+            var model = (CreateApprenticeMemberCommand)createApprenticeModel;
+
+            Assert.That(model, Is.Not.Null);
+
+            model.Should().BeEquivalentTo(createApprenticeModel, c => c
+                .Excluding(m => m.Region)
+            );
+
+            Assert.AreEqual(createApprenticeModel.ApprenticeId, model.ApprenticeId);
```

Comment:
The assertions are not required because the equivalence test on line 19 covers them.